pkg/webhooks/kafkacluster_validator.go

```diff
@@ -47,17 +49,42 @@ func (s KafkaClusterValidator) ValidateUpdate(ctx context.Context, oldObj, newOb
 	if fieldErr != nil {
 		allErrs = append(allErrs, fieldErr)
 	}
+
+	// check internal and external listeners
```

Comment:
Same as above, I don't really see the need to leave the comment here, if there is, I would rename the function to be something more explicit, e.g. `checkInternalAndExternalListeners`